cards/condyle-fracture-lateral/condyle-fracture-lateral

```diff
@@ -0,0 +1,60 @@
+---
+
+title: Elbow Injuries - Condyle Fracture Lateral
+
+authors:
+   - Melody Glenn, MD
+   - Andrew Little, DO
+   - Mary Haas, MD
+Created: 2016/12/12
+updates: null
+Categories:
+   - Peer Reviewed
+   - Orthopedics
+
+---
+
+# Elbow Injuries: Condyle Fracture Lateral
+
+![](Image 1)
+
+## Xray Views
+
+Elbow XR: AP, lateral, internal oblique
+
+## Acute Management
+
+Long-arm posterior splint with elbow at 90° flexion and forearm supination
+
+Consider a long arm cast instead
+
+Operative indications:
+   - >2 mm displacement
+   - Any joint incongruity
+   - Fracture non-union
+
+## Follow-up Timing
+
+Non-operative: Follow up in 5-7 days
+
+Operative: Follow up in 2-3 days
+
+## Notes
+
+Milch Classification:
+   - Type I: Fx line is lateral to trochlear grove
+   - Type II: Fx line into trochlear groove
+
+Complications: Ulnar nerve injury and avascular necrosis
+   - Usually in children 5-10 years old
+   - Majority will require operative stabilization
+
+## References
+
+1. [Wheeless’ Textbook of Orthopaedics] (http://Wheelessonline.com)
```

Comment:
References should be an unordered list: dashes, not numbers. I need to change the documentation to make this clear. There are other existing cards that need this fixed too.